api/v1alpha2/types.go

```diff
@@ -273,14 +273,17 @@ type LoadBalancer struct {
 	*/
 }
 
-// LoadBalancerSKU enumerates the values for load balancer sku name.
+// SKU enumerates the values for load balancer sku name.
```

Comment:
Any wat we can exclude the v1alpha2 types from this check? I'm leaning toward not modifying them at this point since we're about to remove them completely with v1alpha4

Reply:
Sure, I can add a pattern to `exclude-rules` in the config so `golint` skips v1alpha2 types.